runtime-watcher/tests/e2e/utils/utils.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/client-go/discovery"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	machineryruntime "k8s.io/apimachinery/pkg/runtime"
+)
+
+func NewKymaWithSyncLabel(name, namespace, channel, syncStrategy string) *v1beta2.Kyma {
+	return NewKymaBuilder().
+		WithNamePrefix(name).
+		WithNamespace(namespace).
+		WithAnnotation("skr-domain", "example.domain.com").
+		WithAnnotation(shared.SyncStrategyAnnotation, syncStrategy).
+		WithLabel(shared.InstanceIDLabel, "test-instance").
+		WithLabel(shared.SyncLabel, shared.EnableLabelValue).
+		WithChannel(channel).
+		Build()
+}
+
+func IsNotFound(err error) bool {
+	if err == nil {
+		return false
+	}
+	if machineryruntime.IsNotRegisteredError(err) ||
+		meta.IsNoMatchError(err) ||
+		apierrors.IsNotFound(err) {
+		return true
+	}
+	groupErr := &discovery.ErrGroupDiscoveryFailed{}
+	if errors.As(err, &groupErr) {
+		for _, err := range groupErr.Groups {
+			if apierrors.IsNotFound(err) {
+				return true
+			}
+		}
+	}
+	for _, msg := range []string{
+		"failed to get restmapping",
```

Comment:
Will investigate this after this PR. Since most the e2e test code is carried from lifecycle-manager, makes sense to check there first since this util func is heavily used.

Reply:
So unfortunately the first string here is created as error like this
```
if err != nil {
 return false, fmt.Errorf("failed to get restmapping: %w", err)
}
```
in [controller-runtime's apimachinery](https://github.com/kubernetes-sigs/controller-runtime/blob/984a121dbb8f6273eef55ebb4016e3b3deef1919/pkg/client/apiutil/apimachinery.go#L77).
So that error can't be imported.